client/src/consts/theme.tsx

```diff
@@ -1,9 +1,15 @@
 const colors = {
+  gray60: " #6b6b6b",
+  gray80: " #343434",
   gray100: "#393838",
 };
 
 const theme = {
   button: {
     backgroundColor: colors.gray100,
   },
+  menu: {
+    backgroundColor: colors.gray80,
+    borders: colors.gray60,
```

Comment:
Maybe a good idea is to have the names be completely correspondant to the property to which they'll be used.

For eg.
`border-color` would be used from `menu.borderColor`

```suggestion
    borderColor: colors.gray60,
```